lib/src/utils.dart

```diff
@@ -233,11 +280,12 @@ class Utils {
       if (i >= videoRids.length) {
         return;
       }
+      final size = dimensions.min();
       final rid = videoRids[i];
 
       result.add(e.encoding.toRTCRtpEncoding(
         rid: rid,
-        scaleResolutionDownBy: findEvenScaleDownBy(dimensions, e.dimensions),
+        scaleResolutionDownBy: size / e.dimensions.min(),
```

Comment:
Only realizing this now, but maybe we should enforce a minimum here, so that `scaleResolutionDownBy` never gets smaller than 1.0 ?

Reply:
sure!

Reply:
@davidliu I have removed findEvenScaleDownBy for now since it looks ok with M97, what do you think ?

Reply:
Yeah, that should be fine.